Knoema.Client/Client.cs

```diff
@@ -36,22 +38,29 @@ public class Client
 
 		public int HttpTimeout { get; set; }
 
-		private Client()
+		private Client(bool ignoreCertErrors, string scheme)
 		{
 			HttpTimeout = DefaultHttpTimeout;
+
+			_scheme = scheme;
+			if (string.IsNullOrEmpty(_scheme))
+				_scheme = Uri.UriSchemeHttp;
+
+			if (ignoreCertErrors)
```

Comment:
parameter 'ignoreCertErrors' has wrong meaning,
it will disable certificate errors for all connections, not only for this client

Reply:
done